### PR TITLE
**do not use** MAID-2328: enter key used to continue through account creation screens

### PR DESCRIPTION
**Pull request by hunterlester**

### CODE REVIEW

File: src/app/components/create_account.js [hunk 1]
```diff
@@ -106,6 +107,8 @@ export default class CreateAccount extends Component {
         this.inviteCode.value = this.props.inviteCode;
       }
       this.inviteCode.focus();
+    } else if (this.props.navPos === CONSTANTS.CREATE_ACC_NAV.WELCOME) {
```
Comment (shankar2105) [inline src/app/components/create_account.js]: Can we move this block into `componentDidLoad()`??

File: src/app/components/create_account.js [hunk 2]
```diff
@@ -196,28 +199,25 @@ export default class CreateAccount extends Component {
                     }}
                   >Claim an Invitation</button>
                 </div>
+                <div className="auth-f no-border">
```
Comment (shankar2105) [inline src/app/components/create_account.js]: Don't change the HTML structure. This changes the design proposed.

File: src/ffi/ipc.js [hunk 3]
```diff
@@ -235,6 +243,8 @@ const init = () => {
   ipcMain.on('registerSharedMDataDecision', onSharedMDataDecision);
   ipcMain.on('registerOnReqError', onReqError);
   ipcMain.on('skipAuthRequest', skipAuthReq);
+  ipcMain.on('sendRenderer', requestMainResource);
+  ipcMain.on('winstonLogger', winstonLogger);
```
Comment (shankar2105) [inline src/ffi/ipc.js]: We have not implemented winston logging in authenticator, but why to have this IPC listener?

Comment (hunterlester) [reply src/ffi/ipc.js]: @shankar2105 Thank you for catching this. I must have mistakenly merged that winston work into my master branch.

**Pull request closed without merging**